pypeit/par/pypeitpar.py

```diff
@@ -806,9 +813,15 @@ def validate(self):
             return
 
         # Check the frame exists
-        if not os.path.isfile(self.data['pixelflat_file']):
-            raise ValueError('Provided frame file name does not exist: {0}'.format(
-                                self.data['pixelflat_file']))
+        # only the file name is provided, so we need to check if the file exists
+        # in the right place (data/static_calibs/<spectrograph_name>/)
+        find_it = list(dataPaths.static_calibs.glob(f'*/{self.data["pixelflat_file"]}'))
+        if len(find_it) == 0:
+            # check if it is cached
+            cached = cache.search_cache(self.data['pixelflat_file'])
+            if len(cached) == 0:
+                raise ValueError(f'Provided pixelflat file: {self.data["pixelflat_file"]} not found '
+                                 f'in {str(dataPaths.static_calibs.path)}/<spectrograph_name>/ not in the cache')
```

Comment:
I would use the `get_file_path` function instead.  It checks if the file is local, in the cache, or downloadable (and will download it, if it is).
```
try:
    flat_file = dataPaths.static_calibs.get_file_path(self.data['pixelflat_file'])
except PypeItError as e:
    msgs.error(f'Provided pixelflat file, {self.data["pixelflat_file"]}, is not a direct path, '
               'a cached file, or a file that can be downloaded from a PypeIt repository.')
```